woocommerce/woocommerce
# Product Block Editor: autofocus attributes combobox component

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

This PR introduces the `autoFocus` property to the `<AttributesComboboxControl />`, which is used by the New Attributes modal to focus the instance when right after it shows up.





Closes # .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Enable the new Product Block Editor
2. Go to the Variations Tab
3. Click on the `Add options`
4. Confirm that it always focuses in the Attribute combobox instance

https://github.com/woocommerce/woocommerce/assets/77539/a6f68023-af33-4b4d-ae94-5974aef19dca




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
